tools/build_standard_commutations.py

```diff
@@ -21,8 +21,75 @@
 from typing import List
 from qiskit.circuit import Gate, CommutationChecker
 import qiskit.circuit.library.standard_gates as stdg
+from qiskit.circuit.library import PauliGate
 from qiskit.dagcircuit import DAGOpNode
 
+SUPPORTED_ROTATIONS = {
+    "rxx": PauliGate("XX"),
+    "ryy": PauliGate("YY"),
+    "rzz": PauliGate("ZZ"),
+    "rzx": PauliGate("XZ"),
+}
+
+
+@lru_cache(maxsize=10**3)
```

Comment:
This and the functions below are not needed right? This is probably a merge artifact?